gdbflash.sh

```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
 
-cd $1/build
+if [ $# -ne 2 ]; then
+    echo "Please enter two arguments (Project name and Port number)"
```

Comment:
We should also echo an example usage.

Reply:
Also, I think instead of entering just the port number, it might be more intuitive to enter the path, e.g. `/dev/ttyACM0`. Or to provide both! Thoughts?

Reply:
Lastly, we should make it possible to enter the arguments in any order, via flags. For example something like `./gdbflash.sh -b <black-magic-path> -p <project> `